packages/react-async/src/useAsync.js

```diff
@@ -98,6 +98,8 @@ const useAsync = (arg1, arg2) => {
     isMounted.current && dispatch({ type: actionTypes.cancel, meta: getMeta() })
   }
 
+  const reinitialize = () => dispatch({ type: actionTypes.reinitialize, meta: getMeta() })
```

Comment:
```suggestion
  const reinitialize = () => dispatch({ type: actionTypes.reinitialize, payload: options })
```

Reply:
The same as I mentioned above which is related to `Uncaught TypeError: Cannot read property 'debugLabel' of undefined`

Reply:
You're right. Good work 👍